src/components/admin/Dialogs/UserInfo.js

```diff
@@ -79,7 +124,7 @@ const UserInfo = (props: PropsType): React.Node => {
                     </TableRow>
                   </TableHead>
                   <TableBody>
-                    {user.groups.map((group: any): React.Node => (
+                    {groups.map((group: any): React.Node => (
```

Comment:
Possible future improvement: Maybe we can create a type in FlowTypes for that? This is also in other parts of project, but we have to set up communication with gSuite first.

Reply:
yes, but that's a story for another PR